swift/ios-shoppe-demo/ModelControllers/OrderManger.swift

```diff
@@ -0,0 +1,48 @@
+//
+//  OrderManger.swift
+//  ios-shoppe-demo
+//
+//  Created on 4/1/20.
+//  Copyright © 2020 FullSTory All rights reserved.
+//
+
+import Foundation
+
+class OrderManager {
+    static var sharedInstance = OrderManager()
+
+    var currentOrder: Order?
```

Comment:
Does swift instantiate this automatically?

Reply:
thats a good question, I'm not sure I can answer that directly, but its nil until I actually set it equal to something